docs/getting-started.md

```
@@ -0,0 +1,55 @@
+
+This section covers prerequisites, installation, and common recipes for getting started with ai-rizz quickly.
+
+## Prerequisites
+
+- git
+
+## Installation
+
+Install the tool:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)